pages/common/if.md

```diff
@@ -0,0 +1,11 @@
+# if
+
+> Simple shell conditional
+
+- Perform a command depending on another command's success.
+
+`{{cmd}} && echo "success" || echo "failure"``
```

Comment:
It could be even better to make it clear that "another comand" refers to {{cmd}}, since the order is reversed in the description and the actual example. And in any case, there's no need to compact "command" to "cmd" here. Long gone are the times that got us [umount instead of unmount](http://unix.stackexchange.com/questions/9832/why-is-umount-not-spelled-unmount) ;)